### PR TITLE
RTL4b

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: ablySpec/RealtimeClientChannel.swift [hunk 1]
```diff
@@ -13,6 +13,68 @@ class RealtimeClientChannel: QuickSpec {
...
+                        expect(client.connection().state).toEventually(equal(ARTRealtimeConnectionState.Connected), timeout: testTimeout)
+                        let transport = client.transport as! TestProxyTransport
+                        transport.actionsIgnored += [.Closed]
+
+                        let channel = client.channel("test")
+
+                        client.close()
+                        expect(client.connection().state).to(equal(ARTRealtimeConnectionState.Closing))
+
+                        expect(channel.attach()).toNot(beNil())
```
Comment (mattheworiordan) [inline ablySpec/RealtimeClientChannel.swift]: This is not clear that this is an error

**Comment by mattheworiordan** [comment]
Looks good other than the one comment

**Comment by tcard** [comment]
LGTM

File: ablySpec/RealtimeClientChannel.swift (hunk 1 above)
Comment (ricardopereira) [inline ablySpec/RealtimeClientChannel.swift]: Can I correct this after the API changes?

**Comment by ricardopereira** [comment]
Rebased with no changes.

**Comment by tcard** [comment]
This can be merged, right?

File: ablySpec/RealtimeClientChannel.swift (hunk 1 above)
Comment (mattheworiordan) [inline ablySpec/RealtimeClientChannel.swift]: Ok, please add a TODO or an issue though so we don't forget

**Comment by mattheworiordan** [comment]
Sure, merge away

**Pull request merged**